README.md

```diff
@@ -109,9 +109,21 @@ To build the documentation for the SDK, it is being automatically generated with
     cd splunk
     mvn javadoc:javadoc
 
+###SSL Certificate Verification
+SSL Certificate validation is turned ON by default in Splunk Java SDK. Set SSL Certificate as shown below.
+```java
+    HttpService.setSSLCert(<byte[] sslCert>);
```

Comment:
This still needs the data input app to provide the cert. How do the data input app figure out which cert to provide here when communicating with the Splunk Enterprise/Cloud ?